src/aks-preview/azext_aks_preview/_format.py

```diff
@@ -88,6 +88,7 @@ def _aks_table_format(result):
         location: location,
         resourceGroup: resourceGroup,
         kubernetesVersion: kubernetesVersion,
+        currentKubernetesVersion: currentKubernetesVersion,
```

Comment:
Just curious, what's the difference between `kubernetesVersion` and `currentKubernetesVersion`?

Reply:
For 20211101preview api, we have a new feature "Alias Minor Version", where customers can input "1.19" for kubernetesVersion. currentKubernetesVersion will be the exact kubernetesVersion we provision for them, like "1.19.7". If kubernetesVersion is a specific version, then both properties value are the same.